src/Application/Characters/Queries/GetTopUserCharactersByCompetitiveRatingQuery.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AutoMapper;
+using Crpg.Application.Characters.Models;
+using Crpg.Application.Common.Interfaces;
+using Crpg.Application.Common.Mediator;
+using Crpg.Application.Common.Results;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+
+namespace Crpg.Application.Characters.Queries;
+
+public record GetTopUserCharactersByCompetitiveRatingQuery : IMediatorRequest<IList<CharacterCompetitiveRatingViewModel>>
```

Comment:
You'll need to return more info than what's in `CharacterCompetitiveRatingViewModel` to build a leaderboard in the UI.